src/sentry/migrations/0110_sentry_app_creator_backill.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-10-06 17:57
+from __future__ import unicode_literals
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+from django.db import migrations
+
+def backfill_one(audit_log_entry, SentryApp):
+    name = audit_log_entry.data.get("sentry_app")
+    if name:
+        # find the sentry app for that org, name, that's not deleted
+        sentry_app = SentryApp.objects.filter(
+            name=name,
+            owner=audit_log_entry.organization,
+            date_deleted__isnull=True,
+            creator_user__isnull=True,
+        ).first()
```

Comment:
Please don't use `.first()` ever. It's not what you want and will make the query slower.